package.json

```diff
@@ -28,6 +29,7 @@
     "flexsearch": "^0.7.31",
     "glob": "^8.1.0",
     "gray-matter": "^4.0.3",
+    "linkinator": "^5.0.2",
```

Comment:
I chose [linkinator](https://www.npmjs.com/package/linkinator) over [hyperlink](https://www.npmjs.com/package/hyperlink) because:
- It's actively maintained (1 month last publish vs 2 years)
- It's more popular (76,174 vs 16,758 weekly downloads)
- The documentation is better IMO
- The output when running locally is also way nicer 🔥

Reply:
Will follow up with an ADR on this 👂